# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../vdr-streamdev/pid-mask.patch
# Copyright (C) 2009 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

--- ./client/filter.c.orig	2008-03-31 10:38:01.000000000 +0000
+++ ./client/filter.c	2009-05-03 16:49:25.000000000 +0000
@@ -225,7 +225,7 @@
 	while (Running()) {
 		const uchar *block = m_TSBuffer->Get();
 		if (block) {
-			u_short pid = (((u_short)block[1] & PID_MASK_HI) << 8) | block[2];
+			u_short pid = (((u_short)block[1] & TS_PID_MASK_HI) << 8) | block[2];
 			u_char tid = block[3];
 			bool Pusi = block[1] & 0x40;
 			int len = block[4];
--- ./remux/tsremux.c.orig	2008-03-31 10:38:03.000000000 +0000
+++ ./remux/tsremux.c	2009-05-03 16:49:43.000000000 +0000
@@ -20,7 +20,7 @@
 
 int cTSRemux::GetPid(const uchar *Data)
 {
-  return (((uint16_t)Data[0] & PID_MASK_HI) << 8) | (Data[1] & 0xFF);
+  return (((uint16_t)Data[0] & TS_PID_MASK_HI) << 8) | (Data[1] & 0xFF);
 }
 
 int cTSRemux::GetPacketLength(const uchar *Data, int Count, int Offset)
